sympy/polys/densearith.py

```diff
@@ -317,10 +317,7 @@ def dup_quo_ground(f, c, K):
     if not f:
         return f
 
-    if K.is_Field:
-        return [ K.quo(cf, c) for cf in f ]
-    else:
-        return [ cf // c for cf in f ]
+    return [ K.quo(cf, c) for cf in f ]
```

Comment:
I'm not sure about this change. I think that the use of `//` here is for performance with `int/mpz`. Probably it should be limited to that case rather than gated in `is_Field`.